internal/app/app.go

```diff
@@ -137,7 +137,7 @@ func (hs *HTTPService) toMachineService() *api.MachineService {
 }
 
 func (s *Service) toMachineService() *api.MachineService {
-	checks := make([]api.Check, len(s.TCPChecks)+len(s.HTTPChecks))
+	checks := make([]api.Check, 0)
```

Comment:
Oh, my mistake, the intention was to create an empty slice with the expected capacity

```
checks := make([]api.Check, 0, len(s.TCPChecks)+len(s.HTTPChecks))
```

Reply:
Oh that makes sense. Thanks!